src/openapi.json

```diff
@@ -28,6 +28,16 @@
         "name": "_count",
         "required": false,
         "description": "The maximum number of items that should appear per page or request."
+      },
+      "searchParam": {
+        "schema": {
+          "type": "string",
+          "default": ""
+        },
+        "in": "query",
+        "name": "_q",
```

Comment:
I just thought to actually look at FHIR (whoops for not doing this sooner) -- They reference `_search` [in their docs](https://www.hl7.org/fhir/search.html#searchinputs) though it's an indirect reference.  Anyway, I'd probably swap to that.

Reply:
`_content` it is!